.rubocop.yml

```diff
@@ -54,3 +54,6 @@ Rails/CreateTableWithTimestamps:
 
 Rails/FilePath:
   EnforcedStyle: slashes
+
+Style/HashEachMethods:
```

Comment:
Could we put a comment here to say why this was disabled (as per your commit)?

Reply:
Done :)